common/datasets/tedlium2/download.py

```diff
@@ -0,0 +1,32 @@
+import os
+from functools import lru_cache
+from typing import Any, Dict
+
+from i6_core.datasets.tedlium2 import (
+    DownloadTEDLIUM2CorpusJob,
+    CreateTEDLIUM2BlissCorpusJob,
+)
+
+
+@lru_cache()
+def download_data_dict(output_prefix: str = "datasets") -> Dict[str, Any]:
+    data_dict = {}
```

Comment:
I would create a `dataclass` object for this, otherwise it is not verbose at all what the content is.